scripts/installer.py

```diff
@@ -58,14 +59,17 @@ def executeCommand(self, command, shell=False, allow_failure=False) -> tuple[str
                 stderr=subprocess.PIPE
             )
             stdout, stderr = process.communicate()
+            try:
 
-            if stdout is not None and len(stdout) > 0:
-                logging.debug(f'Command output: {stdout.decode(locale.getpreferredencoding(False))}')
+                if stdout is not None and len(stdout) > 0:
+                    logging.debug(f'Command output: {stdout.decode(locale.getpreferredencoding(False))}')
 
-            if allow_failure == False and process.returncode != 0:
-                self.printErrorAndExit(f"Command failed with error : {stderr.decode(locale.getpreferredencoding(False))}")
+                if allow_failure == False and process.returncode != 0:
+                    self.printErrorAndExit(f"Command failed with error : {stderr.decode(locale.getpreferredencoding(False))}")
 
-            return (stdout.decode(locale.getpreferredencoding(False)), stderr.decode(locale.getpreferredencoding(False)))
+                return (stdout.decode(locale.getpreferredencoding(False)), stderr.decode(locale.getpreferredencoding(False)))
+            except :
+                return(stdout, stderr)
```

Comment:
the bug is in the decoding of std out and str err?

Reply:
it was just for stderr in this case